tests/sandbox/test_create_referral.py

```diff
@@ -0,0 +1,60 @@
+from typing import Callable, Dict, Iterable
+
+import pytest
+import asserts
+
+from SandboxTest import SandboxTest
+from data import Actor
+from utils import HttpMethod
+from requests import Response
+
+
+@pytest.mark.sandbox
+class TestCreateReferral(SandboxTest):
+    @pytest.fixture
+    def unauthorised_actors(self) -> Iterable[Actor]:
+        return [Actor.SPC, Actor.SPCA]
+
+    @pytest.fixture
+    def call_endpoint(
+        self,
+        send_rest_request: Callable[[HttpMethod, str, Actor], Response],
+        load_json: Callable[[str], Dict[str, str]],
```

Comment:
load_json defined above as Callable[[str], str]

Reply:
Good spot, corrected this now :).